docs/development.rst

```diff
@@ -0,0 +1,198 @@
+    and the verb SHOULD be something that the object does.
+    The verb MAY be prefixed with "now".
+  * When the changes touch user interface,
+    the grammatical subject of the sentence SHOULD identify the part of the user interface affected
+    and the verb SHOULD be something that the user interface does.
+  * For bugfix,
+    it MAY contain a reference to the version in which the bug was introduced.
+
+Fragment files MUST be placed in ``newsfragments`` and be named ``<ticket number>.<fragment type>``.
+
+Backwards Incompatible Changes
+``````````````````````````````
+
+The fragment type is ``incompat``.
+Here are some examples.
+
+::
+
+   ``magic-folder create`` now requires the ``--path`` option.
+
+::
+
+   Files in a Magic Folder with a name beginning with ``.`` are now uploaded instead of ignored.
+
+Features
+````````
+
+The fragment type is ``feature``.
+Here are some examples.
+
+::
+
+   The downloader can now detect three-party conflicts.
+
+::
+
+   On Windows, the uploader now notices file changes immediately instead of after a several minute delay.
+
+Bug Fixes
+`````````
+
+The fragment type is ``bugfix``.
+Here are some examples.
+
+::
+
+   The uploader no longer fails to upload all files with a space in their name.
+
+::
+
+   The downloader now writes all files beneath the Magic Folder directory instead of the user's home directory.
+
+Dependency/Installation Changes
+```````````````````````````````
+
+The fragment type is ``installation``.
+Here are some examples.
+
+::
+
+   magic-folder now requires Python 3.8 or newer.
+
+::
+
+   magic-folder now requires pynacl.
+
+Configuration Changes
+`````````````````````
+
+The fragment type is ``configuration``.
+
+Removed Features
+````````````````
+
+These are changes which deprecate or remove a feature.
+The fragment type is ``removed``.
+
+Here are some examples.
+
+::
+
+   All HTTP API endpoints beneath ``/v1`` are now deprecated in favor of the ``/v2`` endpoints.
+
+::
+
+   The HTTP API, deprecated since v1.2.3 in favor of the OTP API, has been removed.
+
+
+Other Changes
+`````````````
+
+These are changes that don't easily fit into another category.
+The fragment type is ``other``.
+
+
+Misc/Other
+``````````
+
+The fragment type is ``minor``.
+
+These fragments should be empty.
+Their contents will not be included in the assembled news file.
+
+.. _review the changes:
+
+Reviewing Changes
+-----------------
+
+First and foremost,
+the reviewer's job is to ensure the objective of the corresponding ticket has been satisfied.
+
+Some specific areas to which a reviewer can pay attention:
+
+  * Is the implementation unnecessarily difficult for a human reader to understand
+    (and maintain)?
+  * Does the test suite make correct assertions about the behavior of the code under test?
+  * Does the documentation (developer- and user-facing) accurately describe the new behavior?
+
```

Comment:
This list too will be rendered within a blockquote because of leading spaces, as in:

https://github.com/LeastAuthority/magic-folder/blob/164.dev-docs-etc/docs/development.rst#reviewing-changes